comment/templatetags/comment_tags.py

```diff
@@ -82,9 +84,15 @@ def render_comments(obj, request, oauth=False):
 register.inclusion_tag('comment/base.html')(render_comments)
 
 
-def render_content(comment, number):
-    number = int(number)
-    content = comment.content
+def render_content(comment, number=None):
+    try:
+        number = int(number)
+    except (ValueError, TypeError):
+        number = get_wrapped_words_number()
+
+    # Restrict 2 or more line breaks to 2 <br>
+    content = re.sub(r'(.*)(\n){2,}(.*)', r'\1<br><br>\3', comment.content)
+    content = re.sub(r'(.*)(\n)(.*)', r'\1<br>\3', content)
```

Comment:
This will increase the time considerably because the same regular expression will be compiled again and again for all comments. Instead, you can compile the regular expression at the top of the module and use it here.
```suggestion
# use this at the top of the module
SINGLE_NEW_LINE_RE =  re.compile(r'(.*)(\n){2,}(.*)')
MORE_THAN_ONE_NEW_LINE_RE = re.compile(r'(.*)(\n)(.*)')

content = SINGLE_NEW_LINE_RE.sub(r'\1<br><br>\3', comment.content)
content = MORE_THAN_ONE_NEW_LINE_RE.sub(r'\1<br>\3', content)

```